src/test/scala/ironoxide/FullIntegrationTest.scala

```diff
@@ -50,6 +50,9 @@ class FullIntegrationTest extends AsyncWordSpec with Matchers with EitherValues
       primaryTestUserSigningPrivateKeyBytes
     )
 
+  val javaSdk = jsdk.IronSdk.initialize(deviceContext.toJava[IO].unsafeRunSync)
```

Comment:
The more idiomatic way to do this would be to `val deviceContext.toJava[IO].map(d =>IronSdkSync[IO](jsdk.IronSdk.initialize(d))).unsafeRunSync`

Reply:
Changed.